### DISCUSSION TITLE
How can I enable guicursor blinking?

### DISCUSSION THREAD

**Discussion by JuanZoran** [folke/noice.nvim#420] [General]
I am using wezterm a terminal which supports cursor blink animation, I really enjoy this.
But I find that when I use nvim, it will not work anymore and I found an option named 'guicursor'

When I do this command '=vim.opt.guicursor', I get these outputs: 
`  _value = "a:NoiceHiddenCursor",`, I wonder to know how can I change this option to enable wezterm cursor animation...

**Comment by hopezh** [comment]
Can you try the following? 

```
-- change cursor using nvim's guicursor ---------------------------------------- 
-- [:help guicursor]
vim.cmd([[
    set guicursor=n-v-c:block,i-ci-ve:ver50,r-cr:hor50,o:hor50
      \,a:blinkwait30-blinkoff30-blinkon30-Cursor/lCursor
      \,sm:block-blinkwait175-blinkoff150-blinkon175F
]])
```

**Reply by JuanZoran** [reply]
Cool! It works, but I don't know if this will cause anything weird. why Noice set that options
